src/app/layouts/default/header/menus/main-menu/login-menu/login-menu.tsx

```diff
@@ -7,20 +7,14 @@ import linkHelper from '~/helpers/link';
 function LoginLink() {
     // It's not used directly, but loginLink changes when it does
     useLocation();
-    const addressHinkyQAIssue = React.useCallback(
```

Comment:
This was a very old kludge and TS said it was not legal. I saw no problems arise from removing it.